kv-components/vue/KvProgressBar.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+	<div
+		role="progressbar"
+		aria-label="progress meter"
+		class="h-1 w-full rounded-full bg-gray-300 bg-opacity-low overflow-hidden"
+	>
+		<div
+			ref="progressBarRef"
+			:max="max"
+			:value="value"
+			class="h-1 rounded-full transition-all duration-1000 origin-left ease-in bg-brand"
+			:style="{width: loaded ? `${value}%` : '0' }"
+			:aria-valuemax="value"
+			:aria-valuenow="max"
```

Comment:
It looks like valuemax and valuenow are swapped

Reply:
Good catch, thank you, I've corrected this in my latest commit.